lib/ui/home.dart

```diff
@@ -34,52 +59,79 @@ class _HomeState extends State<Home> {
-                icon: Image.asset(Assets.BOTTOM_BAR_DISCOVER_UNSELECTED, height: bottomBarIconSize),
-                title: Text(Strings.discoverTab),
-              ),
-              BottomNavigationBarItem(
-                activeIcon: Image.asset(Assets.BOTTOM_BAR_COMMUNITY_SELECTED, height: bottomBarIconSize),
-                icon: Image.asset(Assets.BOTTOM_BAR_COMMUNITY_UNSELECTED, height: bottomBarIconSize),
-                title: Text(Strings.communityTab),
-              ),
-            ],
-          ),
-        ));
+      appBar:
+          CustomAppBar.build(viewModel.currentTab, viewModel.goToPrivacyPolicy),
+      // We could share a list of pre defined actions for the app bar.
+      body: _children[viewModel.currentTab],
+      bottomNavigationBar: Theme(
+        data: Theme.of(context).copyWith(primaryColor: Color(primaryGreen)),
+        child: BottomNavigationBar(
+          type: BottomNavigationBarType.fixed,
+          onTap: viewModel.changeTab,
+          currentIndex: viewModel.currentTab,
+          items: [
+            BottomNavigationBarItem(
+              activeIcon: Image.asset(Assets.BOTTOM_BAR_MY_PLOT_SELECTED,
+                  height: bottomBarIconSize),
+              icon: Image.asset(Assets.BOTTOM_BAR_MY_PLOT_UNSELECTED,
+                  height: bottomBarIconSize),
+              title: Text(Strings.myPlotTab),
+            ),
+            BottomNavigationBarItem(
+              activeIcon: Image.asset(Assets.BOTTOM_BAR_PROFIT_LOSS_SELECTED,
+                  height: bottomBarIconSize),
+              icon: Image.asset(Assets.BOTTOM_BAR_PROFIT_LOSS_UNSELECTED,
+                  height: bottomBarIconSize),
+              title: Text(Strings.profitLossTab),
+            ),
+            BottomNavigationBarItem(
+              activeIcon: Image.asset(Assets.BOTTOM_BAR_DISCOVER_SELECTED,
+                  height: bottomBarIconSize),
+              icon: Image.asset(Assets.BOTTOM_BAR_DISCOVER_UNSELECTED,
+                  height: bottomBarIconSize),
+              title: Text(Strings.discoverTab),
+            ),
+            BottomNavigationBarItem(
+              activeIcon: Image.asset(Assets.BOTTOM_BAR_COMMUNITY_SELECTED,
+                  height: bottomBarIconSize),
+              icon: Image.asset(Assets.BOTTOM_BAR_COMMUNITY_UNSELECTED,
+                  height: bottomBarIconSize),
+              title: Text(Strings.communityTab),
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+
+  Future<void> _retrieveDynamicLink() async {
+    final PendingDynamicLinkData data =
+        await FirebaseDynamicLinks.instance.retrieveDynamicLink();
+    final Uri deepLink = data?.link;
+    if (deepLink != null) {
+      FireStoreManager fireStoreManager = FireStoreManager.get();
+      //FIXME: Find a better way to get the articleId
+      String articleId = deepLink.query.substring(3, deepLink.query.length);
```

Comment:
If for the moment we are only sharing articles, we should go on this when other feature required  share, because we had some troubles taking all the parameters from the deepLink.

So i think we can check it in the future to no extend this for the Beta1

Reply:
now the FIXME part is changed to use the .queryParameters so we can retrieve the type, the id, or whatever parameter that we have in the DeepLink